modules/network-observability-loki-secret.adoc

```diff
@@ -30,4 +30,4 @@ stringData:
 <1> The installation examples in this documentation use the same namespace, `netobserv`, across all components. You can optionally use a different namespace for the different components
 
 .Verification
-* Once you create the secret, you should see it listed under *Workloads* -> *Secrets* in the web console.
+* Once you create the secret, you should see it listed under *Workloads* -> *Secrets* in the web console.
```

Comment:
```suggestion
* After you create the secret, you can view the secret under *Workloads* -> *Secrets* in the web console.
```

"Should" does not provide 100 % confidence, IMO, that the secret will be there after completing the procedure.